packages/fast-components-react-base/src/dialog/dialog.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import { get } from "lodash-es";
+import Foundation, { HandledProps } from "../foundation";
+import { DialogProps, IDialogHandledProps, IDialogManagedClasses, IDialogUnhandledProps } from "./dialog.props";
+import { IDialogClassNameContract, IManagedClasses } from "@microsoft/fast-components-class-name-contracts-base";
+
+/* tslint:disable-next-line */
+class Dialog extends Foundation<IDialogHandledProps & IManagedClasses<IDialogClassNameContract>,  React.AllHTMLAttributes<HTMLElement>, {}> {
+    public static defaultProps: Partial<IDialogHandledProps> = {
+        contentHeight: "480px",
+        contentWidth: "640px",
+        visible: false
+    };
+
+    protected handledProps: HandledProps<IDialogHandledProps & IManagedClasses<IDialogClassNameContract>> = {
+        describedBy: void 0,
+        label: void 0,
+        labelledBy: void 0,
+        children: void 0,
+        contentWidth: void 0,
+        contentHeight: void 0,
+        modal: void 0,
+        managedClasses: void 0,
+        visible: void 0
+    };
+
+    /**
+     * Renders the component
+     */
+    public render(): React.ReactElement<HTMLDivElement> {
+        return (
+            <div
+                {...this.unhandledProps()}
+                className={this.generateClassNames()}
+                aria-hidden={!this.props.visible}
```

Comment:
do we want to assume that if it is aria-hidden its also visually hidden and set a style attribute? or were you thinking that it should be controlled by classes to give an opportunity for animation?

Reply:
I spent some time thinking about this and I think that by inlining this we take away flexibility of how someone wants to show/hide as well as their ability to easily animate in and out.

Reply:
Sounds good